src/PimApi/PimCatalogApiClient.php

```diff
@@ -69,4 +69,37 @@ private function getPimUrl(): string
 
         return $pimUrl;
     }
+
+    /**
+     * @return array<mixed>
+     */
+    public function getProducts(string $catalogId, int $limit = 10, string $searchAfter = null, string $updated_after = null, string $updated_before = null): array
+    {
+        $pimUrl = $this->getPimUrl();
+
+        $catalogEndpointUrl = "$pimUrl/api/rest/v1/catalogs/$catalogId/products";
+
+        $response = $this->client->request('GET', $catalogEndpointUrl, [
+            'query' => [
+                'search_after' => $searchAfter,
+                'limit' => $limit,
+                'updated_after' => $updated_after,
+                'updated_before' => $updated_before,
+            ],
+        ])->toArray();
+
+        return $response['_embedded']['items'];
+    }
+
+    /**
+     * @return array<mixed>
+     */
+    public function getProduct(string $catalogId, string $uuid): array
```

Comment:
```suggestion
    public function getCatalogProduct(string $catalogId, string $productUuid): array
```

Reply:
I wonder why you suggested getCatalogProduct @tseho ? IMO as we are in the PimCatalogApiClient, it was clear for me.

Reply:
To reduce any confusion, we are in the demo app.
If you read `$client->getProduct()`, unless you have the complete context, it's not clear that you are fetching the product from a catalog endpoint.